ui/components/Flex.tsx

```diff
@@ -28,7 +30,15 @@ const Styled = (component) => styled(component)`
 class Flex extends React.PureComponent<Props> {
   render() {
     const { className, children } = this.props;
```

Comment:
Since you are already using a destructuring assignment here, why not to assign destructure all needed props in order not to use `this.props.` later?

```
const { className, children, onMouseEnter, onMouseLeave } = this.props;
```